### DISCUSSION TITLE
HTTP Error 500

### DISCUSSION THREAD

**Discussion by DaemonHunter08** [ngoduykhanh/PowerDNS-Admin#1590] [Q&A]
Need some help.

I have fresh install debian 11 and follow this guide "https://github.com/PowerDNS-Admin/PowerDNS-Admin/blob/master/docs/wiki/install/Running-PowerDNS-Admin-on-Ubuntu-or-Debian.md". and it install well, however, when trying to manage one of my domains, the page gives me this kind of error "HTTP 500 Error", "Oops! Something went wrong.".

and found this on the logs. "Returned status 500 and content Internal Server Error"

![image](https://github.com/PowerDNS-Admin/PowerDNS-Admin/assets/136132004/9c742e7b-a5a7-43fa-bf23-8a3e7dbd1f5f)

**Comment by AzorianMatt** [comment]
@DaemonHunter08 your PowerDNS server itself is returning a 500 server error from the API. Check that installation as it does not appear to be working correctly.

**Reply by DaemonHunter08** [reply]
with the regards to the API, i'm using PowerDNS built in API feature, and i have remote database backend. Tho, the system works fine with to older version of PowerDNS-Admin.

**Reply by AzorianMatt** [reply]
What version of PDNS are you using? All I can tell you is that the logs say clear as day, a 500 error returned from the PDNS API request made by PDA to load the records.

If this only happens on this particular zone, I recommend sharing everything you can about that zone, so we can get an idea of why this one is different.

**Reply by DaemonHunter08** [reply]
i'm using pdns 4.7.4 and the issue persist on all zone.
